### PR TITLE
dlrm loss reduction

### PR DESCRIPTION
**Pull request by ShawnXuan**
change loss reduction to mean

### CODE REVIEW

File: RecommenderSystems/dlrm/dlrm_train_eval.py [hunk 1]
```diff
@@ -442,8 +442,7 @@ def __init__(
     def build(self, labels, dense_fields, sparse_fields):
         logits = self.module(dense_fields.to("cuda"), sparse_fields.to("cuda"))
         loss = self.loss(logits, labels.to("cuda"))
-        reduce_loss = flow.mean(loss)
-        reduce_loss.backward()
+        loss.backward()
         return reduce_loss.to("cpu")
```
Comment (MARD1NO) [inline RecommenderSystems/dlrm/dlrm_train_eval.py:446]: return loss.to("cpu")

**Review by MARD1NO — APPROVED** [review]
